package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@azure/static-web-apps-cli",
-  "version": "1.0.6",
```

Comment:
Lets validate it once.. should it be alpa?

Reply:
yes, modified it now.